### DISCUSSION TITLE
Problem with webpage name when using this empty element

### DISCUSSION THREAD

**Discussion by abhishekk2604** [h5bp/html5-boilerplate#2838] [Q&A] [answered]
I was learning Html document when this happened.

The empty tag- 
`<meta name="viewport" content="width=device-width"> `
Seems to make some issues with the webpage name which goes under title element.
The webpage name just goes Untitled when i use this tag.
Can someone tell me why?

**Comment by meyshad** [comment, marked as answer]
Add a viewport `<meta>` tag with the appropriate key-value pairs to the `<head>` of your page:

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1">
  </head>
```

**Reply by developer77android-cpu** [reply]
I realize this was four years ago however it is a  part of what makes our projects..responsive !! Required info def.
